R/formatters.R

```diff
@@ -77,6 +80,10 @@ formatter_glue_or_sprintf <- structure(function(msg, ..., .logcall = sys.call(),
     if (length(params) > 0 & length(sprintfparams) == 0) {
         sprintfparams <- seq_along(params)
     }
+    if (is.null(msg) || length(msg) == 0) {
+        msg <- ''
+        # no need to touch params, both glue and sprintf handle missing placeholders
```

Comment:
why do we need to run the rest of this fn?

```suggestion
        return('')
```